### PR TITLE
Run long tests once a day

### LINKED ISSUE #2397: run long tests once a day
https://github.com/PrismarineJS/mineflayer/blob/ad52fcaee0fc0529796f3ed37ea11346b1efdc55/test/externalTest.js#L18

in particular dig everything

### PR DESCRIPTION
**Pull request by imharvol**
Runs all tests every day at 00:00 UTC

Closes  #2397

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -52,3 +54,6 @@ jobs:
         run: npm install
       - name: Start Tests
         run: npm run mocha_test -- -g ${{ matrix.mcVersion }}
+
+    env:
+      RUN_EXCLUDED_TESTS: ${{ github.event.schedule == '0 0 * * *' }}
```
Comment (rom1504) [inline .github/workflows/ci.yml:59]: Let's make it run every hour to make sure it works, then we switch it back to every day

**Comment by rom1504** [comment]
Let's try. Not confident it'll work

**Comment by imharvol** [comment]
Seems to work: https://github.com/PrismarineJS/mineflayer/actions/runs/4092207626

**Comment by rom1504** [comment]
No it failed.

Can you
1. Disable it
2. Solve it
3. Try again

**Comment by rom1504** [comment]
@imharvol can you please do a PR with the revert ?

**Pull request merged**